<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>970</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1020</width>
    <height>970</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OTTRS</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>935</x>
      <y>30</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>109</red>
          <green>109</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>109</red>
          <green>109</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>고도 M</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>30</y>
      <width>911</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpBox1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>1000</width>
      <height>280</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>109</red>
          <green>109</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>109</red>
          <green>109</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>고도 M</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>오늘의 추천영화</string>
    </property>
    <widget class="QLabel" name="lb5">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb4">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUp1">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>20</y>
       <width>21</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>241</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lb1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDown1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>21</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>241</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBox2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>1000</width>
      <height>280</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>109</red>
          <green>109</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>109</red>
          <green>109</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>고도 M</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>어제의 추천영화</string>
    </property>
    <widget class="QLabel" name="lb10">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb9">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb7">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb8">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUp2">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>20</y>
       <width>21</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>241</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDown2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>21</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>241</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBox3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>1000</width>
      <height>280</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>109</red>
          <green>109</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>109</red>
          <green>109</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>고도 M</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>내일의 추천영화</string>
    </property>
    <widget class="QLabel" name="lb15">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb14">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb12">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb13">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDown3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>21</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>241</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUp3">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>30</y>
       <width>21</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>241</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiondfsdf">
   <property name="text">
    <string>dfsdf</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
